packages/ios-poc/src/launchIOS.ts

```diff
@@ -47,7 +47,7 @@ const executeAsyncCommand = (command: string): Promise<void> => {
 };
 
 const startRecord = (simulatorId: string, traceFile: string): Promise<void> => {
-  const templateFilePath = `${__dirname}/../Flashlight.tracetemplate`;
+  const templateFilePath = `${__dirname}/../tp.tracetemplate`;
```

Comment:
should we delete `Flashlight.tracetemplate` then? I liked having Flashlight in the name to be honest :D

Reply:
I will rename the template to have Flashlight again

Reply:
Done